backend/src/pilstark/estark.rs

```diff
@@ -60,14 +60,36 @@ impl<F: FieldElement> BackendImpl<F> for EStark {
 
         let mut pil: PIL = pilstark::json_exporter::export(pil);
 
-        /*
+        // TODO starky requires a fixed column with the equivalent
+        // semantics to Polygon zkEVM's `L1` column.
+        // It takes the name of that column via the API.
+        // Powdr generated PIL will always have `main.first_step`,
+        // but directly given PIL may not have it.
+        // This is a hack to inject such column if it doesn't exist.
+        // It should be eventually improved.
```

Comment:
We could have an optimizer step that merges equivalent columns, but then we would also need to keep a reference to that column, or at least get access to the replacement function.